.github/workflows/trigger_azdo_ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Trigger AzDO CIs
```

Comment:
@tkrebes I moved this trigger stuff into a separate workflow. I think this will be a nice change, but I don't know that I can test it pre-submit. But, at least I can double-check if the azdo pipelines run once it goes in. Or I'm open to other ideas.

Reply:
Oops. Looking at the build results, maybe this doesn't work even on PR builds. I'll try to figure that out.

Reply:
The failure was because I missed a condition in one of the trigger actions, which caused it to try to trigger the workflow. This failed because PR builds do not have access to secrets. I think that's fixed now. And we'll have to see how it works once the change is in main.